src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -85,7 +85,8 @@ class HoldRequest extends React.Component {
             <p>When this item is ready, you will use it in the following location:</p>
 
             <fieldset className="select-location-fieldset">
-              <label className="group selected" htmlFor="location1">
+              <legend className="visuallyHidden">Select a pickup location</legend>
+              <div className="group selected">
```

Comment:
@beefoo the `label` that was here didn't have an appropriate form element. Removed it until we create the radio button form.